src/script/standard.h

```diff
@@ -41,6 +43,11 @@ class CScriptID : public uint160
 };
 
 static const unsigned int MAX_OP_RETURN_RELAY = 8192;      //! bytes
```

Comment:
Why do we still need MAX_OP_RETURN_RELAY (although you have introduced two different consts for kmd and AC)?

Reply:
Actually, it was just a default value, like static initialization. But you are right – since we don't use it anywhere, we can remove it.